dbms/tests/clickhouse-test

```diff
@@ -468,6 +508,7 @@ if __name__ == '__main__':
     group=parser.add_mutually_exclusive_group(required=False)
     group.add_argument('--shard', action='store_true', default=None, dest='shard', help='Run sharding related tests (required to clickhouse-server listen 127.0.0.2 127.0.0.3)')
     group.add_argument('--no-shard', action='store_false', default=None, dest='shard', help='Do not run shard related tests')
+    group.add_argument('--client-option', nargs='+', help='Specify additional client argument')
```

Comment:
Does `nargs='+'` mean that at least one argument is always required?

Reply:
No

Reply:
https://docs.python.org/3/library/argparse.html#nargs
`'+'. Just like '*', all command-line args present are gathered into a list. Additionally, an error message will be generated if there wasn’t at least one command-line argument present.`